### PR TITLE
Update Syntax Highlight Color Scheme

### PR DESCRIPTION
**Pull request by Math3v**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Description
Resolves https://github.com/thepracticaldev/dev.to/issues/1685.

I used http://colorsafe.co/ to generate color scheme based on the existing background color. I tried to pick similar colors, but it's really on everybody's taste to pick the right combination. Anyway, I've extracted the color scheme to a handful of SCSS variables, so it should be pretty easy to re-do the colors.

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
Screenshot 2020-02-27 at 18 26 35
Screenshot 2020-02-27 at 18 26 51
Screenshot 2020-02-27 at 18 26 58
Screenshot 2020-02-27 at 18 27 11

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

Artist

### CODE REVIEW

File: app/assets/stylesheets/syntax.scss [hunk 1]
```diff
@@ -13,214 +13,220 @@ div.inner-comment div.body div.highlight pre.highlight {
...
-  background: #29292e;
-  color: #f8f8f2;
+  background: $code-highlight-background-color;
+  color: $code-highlight-text-color;
 }
 .highlight .c {
-  color: #75715e;
+  color: $code-highlight-comment-color;
 } /* Comment */
-.highlight .err {
```
Comment (ludwiczakpawel) [inline app/assets/stylesheets/syntax.scss]: Any reason for removing `.err` styling?

File: app/assets/stylesheets/syntax.scss [hunk 2]
```diff
@@ -13,214 +13,220 @@ div.inner-comment div.body div.highlight pre.highlight {
   }
 }
 
+$code-highlight-background-color: #29292e;
+$code-highlight-text-color: #89c4f4;
```
Comment (ludwiczakpawel) [inline app/assets/stylesheets/syntax.scss]: This one used to be very light color, like `#f8f8f2`, close to white. Now it's a shade of light blue which makes whole code block looks a bit too colorful IMO.

File: app/assets/stylesheets/syntax.scss [hunk 3]
```diff
@@ -13,214 +13,220 @@ div.inner-comment div.body div.highlight pre.highlight {
...
+$code-highlight-comment-color: #d5d5d5;
```
Comment (ludwiczakpawel) [inline app/assets/stylesheets/syntax.scss]: I think it's common pattern in code editors to make comments color a bit "faded out". Previously it was grey `#75715e` which I think worked pretty well.

File: app/assets/stylesheets/syntax.scss (hunk 1 above)
Comment (Math3v) [reply app/assets/stylesheets/syntax.scss]: @ludwiczakpawel Yes, it's not a good idea to remove error syntax highlighting. I added it back with an updated definition, as the original one was redundant and with same colors it always applies the biggest radius. I made it a little bit smaller as 10px makes it too visible imho.

File: app/assets/stylesheets/syntax.scss (hunk 2 above)
Comment (Math3v) [reply app/assets/stylesheets/syntax.scss]: @ludwiczakpawel Good point. Fixed.

File: app/assets/stylesheets/syntax.scss (hunk 3 above)
Comment (Math3v) [reply app/assets/stylesheets/syntax.scss]: @ludwiczakpawel I cannot use the original color as it has terrible accessibility contrast ratio. I used very similar one, but slightly more visible on the background.

Screenshot 2020-02-29 at 12 18 08

**Comment by Math3v** [comment]
@lisasy Any chance you'll have chance to take a look, please? :)
@ludwiczakpawel Maybe we can ask someone else for a review?

**Pull request merged**